deploy/disk/disk-plugin.yaml

```diff
@@ -45,14 +45,9 @@ spec:
             allowPrivilegeEscalation: true
           image: registry.cn-hangzhou.aliyuncs.com/plugins/csi-diskplugin:***
           args :
-            - "--nodeid=$(NODE_ID)"
```

Comment:
consider NodeId is not always same as Instance ID;

Reply:
Node ID is desinged for CSI plugin to broadcase it's instance ID for later detach/attache usage, as far as I know. It is **not** k8s node name.


Anyway, this PR suggests the correct way we use for NodeID. If you want to inject a NODE ID from environment, it is still OK.

Reply:
Agree, thanks Jerry.